tools/code_format/config.yaml

```diff
@@ -295,3 +295,4 @@ visibility_excludes:
 - source/extensions/transport_sockets/common/BUILD
 - source/extensions/udp_packet_writer/default/BUILD
 - source/extensions/udp_packet_writer/gso/BUILD
+- source/extensions/path/pattern_template_lib/BUILD
```

Comment:
Presumably we don't need this in the PR since it's not yet being used elsewhere?

Reply:
Removed